internal/util/update/resource-merge.go

```diff
@@ -106,6 +101,16 @@ func (u ResourceMergeUpdater) Update(options UpdateOptions) error {
 		return err
 	}
 
+	// perform auto-setters after the package is updated
+	a := setters.AutoSet{
```

Comment:
This seems to be part of the ResourceMerge updater strategy. It seems like this behavior should be the same regardless of the update strategy?

Reply:
Agree. Moved it up to update logic.